### PR TITLE
Implemented employee list/index view

### LINKED ISSUE #7: HR should be able to see a list of employees
## Acceptance Criteria

**Given** an HR employee wants to view employees
**When** the employee clicks on the Employees item in the navigation bar
**Then** all current employees should be listed with the following information

1. First name and last name
1. Department

## Testing Criteria

* Your test suite must verify that the response has an `employees` collection in the context.
* Your test suite must verify that the correct template was used to render the response.
* Your test suite must verify that the content of the response has one, or more, of the values that are expected. For example, if one of the employees in Fred Jackson, then your test must test that both "Fred" and "Jackson" are in the response content. Likewise for the department.

### PR DESCRIPTION
**Pull request by williamkimball**
# Description
Added Employee Index view that shows Employee First name, Employee Last Name, and Department name.
 Fixes #7 
 ## Type of change
 Please delete options that are not relevant.
- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update
 # Testing Instructions

Start the program by cd'ing into the WorkforceManagement and using the command dotnet run. Once the program is running, navigate to your browser and go to the localhost address shown in the command prompt that opens up when you run the program. Click on the Employees tab in the navbar, and you should see a list with First name, Last Name, and department name for the number of employees in your database.
 # Checklist:
 - [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added test instructions that prove my fix is effective or that my feature works

### CODE REVIEW

File: WorkforceManagement/WorkforceManagement/Controllers/EmployeeController.cs [hunk 1]
```diff
@@ -1,18 +1,69 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Threading.Tasks;
-using Microsoft.AspNetCore.Http;
-using Microsoft.AspNetCore.Mvc;
-
+﻿using System;
```
Comment (Shu52) [inline WorkforceManagement/WorkforceManagement/Controllers/EmployeeController.cs]: Please add Author and purpose

File: WorkforceManagement/WorkforceManagement/Views/Employee/Index.cshtml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+<!DOCTYPE html>
+
+<html>
+<head>
+    <meta name="viewport" content="width=device-width" />
+    <title>Index</title>
+</head>
+<body>
+<p>
```
Comment (Shu52) [inline WorkforceManagement/WorkforceManagement/Views/Employee/Index.cshtml]: please add line describing this view

File: WorkforceManagement/WorkforceManagement/Views/Employee/Index.cshtml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+            <td>
+                @Html.DisplayFor(modelItem => item.LastName)
+            </td>
+            <td>
+                @Html.DisplayFor(modelItem => item.Department.DepartmentName)
+            </td>
+            <td>
+                @Html.ActionLink("Edit", "Edit", new { /* id=item.PrimaryKey */ }) |
+                @Html.ActionLink("Details", "Details", new { /* id=item.PrimaryKey */ }) |
+                @Html.ActionLink("Delete", "Delete", new { /* id=item.PrimaryKey */ })
```
Comment (Shu52) [inline WorkforceManagement/WorkforceManagement/Views/Employee/Index.cshtml]: please remove delete link

File: WorkforceManagement/WorkforceManagement/Controllers/EmployeeController.cs (hunk 1 above)
Comment (williamkimball) [reply WorkforceManagement/WorkforceManagement/Controllers/EmployeeController.cs]: Tis done

**Pull request merged**